linked_list/linked_list.py

```diff
@@ -13,83 +12,161 @@ def __init__(self):
 
     # returns the value in the first node
     # returns None if the list is empty
-    # Time Complexity: ?
-    # Space Complexity: ?
+    # Time Complexity: Linear O(N), size of LL

```

Comment:
⏱ Time complexity would actually be O(1) here, because you only ever need to look at the head of the list.